kubernetes/apps/charts/jupyterhub/values.yaml

```diff
@@ -14,6 +14,7 @@ jupyterhub:
       limit: 10G
       # Very roughly I have seen most usage in the 2-3GB range
       guarantee: 3G
+    capacity: 32Gi
```

Comment:
@vevetron it looks like this key should be under a `storage` key: https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/HEAD/jupyterhub/values.yaml#L411

Reply:
Yea I messed up! Fixed now.